quickstart-services.yml

```diff
@@ -2,6 +2,13 @@ version: '3.7'
 
 networks:
   ct_server_dev:
+  ct_dev_net:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 192.168.10.0/24
```

Comment:
hardcoded?

Reply:
Yes, it is a private bridge interface shared between docker and the host.